database/mssql/src/test/java/org/ballerinalang/scenario/test/database/mssql/CallTest.java

```diff
@@ -0,0 +1,250 @@
+import org.ballerinalang.model.values.BByte;
+import org.ballerinalang.model.values.BDecimal;
+import org.ballerinalang.model.values.BFloat;
+import org.ballerinalang.model.values.BInteger;
+import org.ballerinalang.model.values.BValue;
+import org.ballerinalang.model.values.BValueArray;
+import org.ballerinalang.scenario.test.common.ScenarioTestBase;
+import org.ballerinalang.scenario.test.common.database.DatabaseUtil;
+import org.ballerinalang.scenario.test.database.util.AssertionUtil;
+import org.ballerinalang.test.util.BCompileUtil;
+import org.ballerinalang.test.util.BRunUtil;
+import org.ballerinalang.test.util.CompileResult;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Properties;
+
+@Test(groups = Constants.MSSQL_TESTNG_GROUP)
+public class CallTest extends ScenarioTestBase {
+    private CompileResult callCompilerResult;
+    private String jdbcUrl;
+    private String userName;
+    private String password;
+    private Path resourcePath;
+
+    @BeforeClass
+    public void setup() throws Exception {
+        Properties deploymentProperties = getDeploymentProperties();
+        jdbcUrl = deploymentProperties.getProperty(Constants.MSSQL_JDBC_URL_KEY) + ";databaseName=simpledb";
+        userName = deploymentProperties.getProperty(Constants.MSSQL_JDBC_USERNAME_KEY);
+        password = deploymentProperties.getProperty(Constants.MSSQL_JDBC_PASSWORD_KEY);
+
+        ConfigRegistry registry = ConfigRegistry.getInstance();
+        HashMap<String, String> configMap = new HashMap<>(3);
+        configMap.put(Constants.MSSQL_JDBC_URL_KEY, jdbcUrl);
+        configMap.put(Constants.MSSQL_JDBC_USERNAME_KEY, userName);
+        configMap.put(Constants.MSSQL_JDBC_PASSWORD_KEY, password);
+        registry.initRegistry(configMap, null, null);
+
+        resourcePath = Paths.get("src", "test", "resources").toAbsolutePath();
+        DatabaseUtil.executeSqlFile(jdbcUrl, userName, password,
+                Paths.get(resourcePath.toString(), "sql-src", "ddl-select-update-test.sql"));
+        DatabaseUtil.executeSqlFile(jdbcUrl, userName, password,
+                Paths.get(resourcePath.toString(), "sql-src", "ddl-call-test.sql"));
+        DatabaseUtil.executeSqlFile(jdbcUrl, userName, password,
+                Paths.get(resourcePath.toString(), "sql-src", "dml-call-test.sql"));
+        callCompilerResult = BCompileUtil
+                .compile(Paths.get(resourcePath.toString(), "bal-src", "call-test.bal").toString());
+    }
+
+    @Test(description = "Test numeric type IN params")
+    public void testCallInParamNumericTypes() {
+        BValue[] returns = BRunUtil.invoke(callCompilerResult, "testCallInParamNumericTypes");
+        AssertionUtil.assertCallQueryReturnValue(returns[0]);
+    }
+
```

Comment:
Shall we separate numeric types in to following categories 
1. Integer types eg: INT, SMALLINT, BIGINT etc
2. Floating point types eg: FLOAT, DOUBLE etc
3. Fixed point types eg: DECIMAL, NUMERIC etc

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-scenario-tests/pull/38/commits/1402cee58bb3dd0339a86f09a6228ae0e8f64f1f